lib/gems/pending/util/object_storage/miq_swift_storage.rb

```diff
@@ -78,6 +78,26 @@ def upload_single(dest_uri)
     end
   end
 
+  def download_single(source, destination)
+    #
+    # Get the remote path, and parse out the bucket name.
```

Comment:
I don't think we need a comment here

Reply:
Fair enough.  I think I copied this from `s3`, so I was clearly using the wrong terminology as a result.  Will remove.